lib/registries/addon/drafts/draft/-components/register/template.hbs

```diff
@@ -0,0 +1,43 @@
+<OsfButton
+    data-test-goto-register
+    @type='success'
+    @onClick={{action (mut this.showPartialRegDialog) true}}
+    @disabled={{not this.draftManager.registrationResponsesIsValid}}
+>
+    {{t 'registries.drafts.draft.register'}}
+</OsfButton>
+
+{{#unless this.draftManager.registrationResponsesIsValid}}
+    <div data-test-invalid-responses-text class='text-danger'>
+        {{t 'registries.drafts.draft.review.invalid_warning'}}
+    </div>
+{{/unless}}
+
+{{#if this.rootNode}}
+    <Registries::PartialRegistrationModal::Manager
+        @rootNode={{this.rootNode}}
+        as |partialRegistrationManager|
+    >
+        <Registries::PartialRegistrationModal
+            @isOpen={{this.showPartialRegDialog}}
+            @manager={{partialRegistrationManager}}
+            @onContinue={{action this.onContinue}}
+            @onClose={{action (mut this.showPartialRegDialog) false}}
+        />
+    </Registries::PartialRegistrationModal::Manager>
+{{/if}}
+
+{{#if this.registration}}
+    <Registries::FinalizeRegistrationModal::Manager
+        @registration={{this.registration}}
+        @onSubmitRegistration={{action this.onSubmit}}
```

Comment:
Should we make the naming consistent?
```suggestion
        @onSubmitRegistration={{action this.onSubmitRegistration}}
```

Reply:
`onSubmit` wraps `onSubmitRegistration`, which is passed in to this component. Maybe we should call what is passed in something different?

Reply:
I still don't understand why it couldn't be the same name. Makes it easier to track.

Reply:
Oh I see what is going on. `onSubmitRegistration` is passed in from the controller to this component, and then wrapped in `onSubmit`, and then passed to the manger as `onSubmitRegistration` again. This is confusing. Maybe there's a better way to rename the actions.